src/modules/cmdpal/Microsoft.CmdPal.UI.ViewModels/AliasManager.cs

```diff
@@ -86,16 +86,6 @@ public void UpdateAlias(string commandId, CommandAlias? newAlias)
             return;
         }
 
-        // If we already have _this exact alias_, do nothing
-        if (newAlias != null &&
-            _aliases.TryGetValue(newAlias.SearchPrefix, out var existingAlias))
-        {
-            if (existingAlias.CommandId == commandId)
-            {
-                return;
```

Comment:
can you explain a bit why _removing_ this duplicate check fixes duplicated aliases